wgs/workflows/germline_calling_consensus/consensus.py

```diff
@@ -48,7 +48,10 @@ def get_counts(record, caller, sample_id):
         alt = sample['AC']
     elif caller == 'freebayes':
         depth = sample['DP']
-        ref = sample['RO']
+        try:
```

Comment:
why?

is RO missing in vcf in some cases?

what happens later when the ref is referenced. probably undefined var error.


just skipping errors like this is very bad practice